pkg/chunk/util/util.go

```diff
@@ -0,0 +1,77 @@
+package util
```

Comment:
Why the util package instead of another thing in chunk? I think everywhere that uses this right now uses chunk as well

Reply:
I agree util packages suck, but I'm trying to break the chunk package up - its a bit of a behemoth, and I find I regularly get into circular imports with it.  Hence this got stuck in a new package.